spec/index.bs

```diff
@@ -16660,6 +16695,21 @@ expose real values whenever the feature is available on the adapter:
 - New WGSL extensions:
     - [=extension/subgroups=]
 
+<h3 id=dom-gpufeaturename-buffer-map-extended-usages data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"buffer-map-extended-usages"`
+</h3>
+
+Allows the creation of mappable {{GPUBuffer}}s with any other {{GPUBufferUsage}} flags, and allows
+setting both {{GPUMapMode/READ}} and {{GPUMapMode/WRITE}} bits as a valid map mode in
+{{GPUBuffer}}.{{GPUBuffer/mapAsync()}}.
+
+This feature adds the following [=optional API surfaces=]:
+- Allows the use of {{GPUBufferUsage/MAP_READ}} with any other {{GPUBufferUsage}} flags as
+    |descriptor|.{{GPUBufferDescriptor/usage}} in the creation of {{GPUBuffer}}s.
+- Allows the use of {{GPUBufferUsage/MAP_WRITE}} with any other {{GPUBufferUsage}} flags as
+    |descriptor|.{{GPUBufferDescriptor/usage}} in the creation of {{GPUBuffer}}s.
+- Allows setting both {{GPUMapMode/READ}} and {{GPUMapMode/WRITE}} bits in the `mode` parameter of
+    {{GPUBuffer}}.{{GPUBuffer/mapAsync()}}.
```

Comment:
[editorial] This list really says the same thing as the summary. Maybe the summary could just be removed?